packages/chain-bridge/src/adapters/StargateAdapter.sol

```diff
@@ -818,6 +1350,153 @@ contract StargateAdapter is Ownable, IBridgeAdapter, ILayerZeroComposer {
         );
     }
 
+    /**
+     * @dev Try to deposit assets to FleetCommander without reverting on failure
+     */
+    function _tryDepositToFleetCommander(
+        address fleetCommander,
+        address asset,
+        uint256 amount,
+        address shareRecipient,
+        bytes memory referralCode,
+        bytes32 operationId,
+        address, // originalUser
+        address, // sourceAsset
```

Comment:
why is this here if this method is not usedd elswhere ? should the originalUser be validated ?